docs/source/transformers/Translate.rst

```diff
@@ -0,0 +1,141 @@
+.. _Translate:
+
+Translate
+================================
+
+Translate Robot Framework source files from one or many languages to different one.
+
+.. note::
+    Required Robot Framework version: >=6.0
+
+.. |TRANSFORMERNAME| replace:: Translate
+.. include:: disabled_hint.txt
+
+
+Example of Robot Framework markers translation:
+
+.. tab-set::
+
+    .. tab-item:: Before (default English)
+
+        .. code:: robotframework
+
+            *** Settings ***
+            Documentation    Dokumentacja zestawu
+
+            Library    Collections
+            Variables    vars.py
+
+            *** Test Cases ***
+            Pierwszy test
+                [Setup]    Setup Keyword
+                Step 1
+
+    .. tab-item:: After (translated to Polish)
+
+        .. code:: robotframework
+
+
+            *** Ustawienia ***
+            Documentation    Dokumentacja zestawu
+
+            Biblioteka    Collections
+            Zmienne    vars.py
+
+            *** Przypadki Testowe ***
+            Pierwszy test
+                [Inicjalizacja]    Setup Keyword
+                Step 1
+
+The language can be configured using ``language`` parameter with the language code (default ``en`` - English)::
+
+    robotidy --configure Translate:enabled=True:language=se
+
+Since the translation is from one or many languages to one, only one language can be configured.
+
+Source language
+----------------
+
+Robotidy will translate only markers that can be recognized. If your source file is written in different language
+you need to configure Robotidy to recognize given language. See :ref:`language_support` for more details.
+Following example configure Robotidy to read English, Polish and German languages and translate Robot Framework
```

Comment:
I can't see English in the example below. Is it turned on by default?

Reply:
Yes, RF always include English by default